src/utils/task_stats_alloc/Cargo.toml

```
@@ -15,6 +15,10 @@ tokio = { version = "0.2", package = "madsim-tokio", features = [
     "time",
     "signal",
 ] }
-workspace-hack = { path = "../../workspace-hack" }
+#workspace-hack = { path = "../../workspace-hack" }
```

We can either disable it on loom:

```suggestion
[target.'cfg(not(loom))'.dependencies]
workspace-hack = { path = "../../workspace-hack" }
```

or just remove it:

```suggestion
```

Personally I prefer to remove it.